### ISSUE TITLE
Dependencies and env.sh missing..

### ISSUE THREAD

**Issue by Maigre** [urbanlab/griotte#3]
Hi,
i'm currently trying to build and start griotte without success.

> make production 

(as mentionned in the Readme) is not available anymore but i found that i have to use instead

> make _production

but then those packages are missing:
griotte.adc / griotte.gpio / griotte.multimedia / griotte.ws
and the corresponding scripts in the griotte/bin/ directory.

I tried to comment them out from the setup.py files, so the make operation can proceed.

Once it is done, i tried the 

> source griotte/tools/env.sh

but this script is missing too !

i figured that i should maybe use the  griotte/tools/start_all.sh script,
but the tornado module is then marked as unavailable.
(but pip install tornado confirms it is already installed). 
I guess it's about source environnement not configured properly since env.sh is missing...

Could you please guide me to setup and start griotte from the github available sources ?

Thanks :)

Best Regards,
Thomas

**Comment by leucos** [comment]
Hey Thomas. Yup, the instructions are a bit outdated. In fact, I used virtualenv before, but then ditched it for various reasons. I'll check this out ASAP, but I'm a bit busy right now.
TL;DR: don't use virtualenv, install modules directly with pip :

```
pip3 install -r production-requirements.txt
pip3 install -r development-requirements.txt
```

It probably won't solve all your problems, but will certainly take you forward.
Note the project is python3 ! So you have to use `pip3` (might be `pip3.2` or something along the lines, I don't remember exactly).
